Marlin/src/module/settings.cpp

```diff
@@ -1015,9 +1018,14 @@ void MarlinSettings::postprocess() {
       #endif
 
       #if ENABLED(AUTO_BED_LEVELING_BILINEAR)
+        #if ENABLED(VARIABLE_GRID_POINTS)
+          EEPROM_WRITE(bedlevel.nr_grid_points);
```

Comment:
Here you write bedlevel.nr_grid_points. but later in 2041 "points" is loaded.

Trying to adopt your great work ;)

Reply:
Thanks 🙂.

`bedlevel.nr_grid_points` is a variable where the points are stored during runtime and other code can reach there and grab them.
`points` at line 2041 is a intermediary variable because the points cannot be loaded straight to the abovementioned variable but have to be processed by `bedlevel.set_grid(....)`